src/deep-log/lib/Driver/ConsoleDriver.js

```diff
@@ -20,8 +44,70 @@ export class ConsoleDriver extends AbstractDriver {
    * @param {*} context
    */
   log(msg, level, context) {
-    var datetime = AbstractDriver.datetime;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['datetime' is defined but never used.](https://www.codacy.com/app/MitocGroup/deep-framework/file/1307400312/issues/source?bid=2895475&fileBranchId=2951459#l47)